src/es/components/molecules/form/Form.js

```diff
@@ -87,6 +104,15 @@ export default class Form extends Shadow() {
     return !this.root.querySelector(`:host > style[_css], ${this.tagName} > style[_css]`)
   }
 
+  /**
+   * evaluates if a render is necessary
+   *
+   * @return {boolean}
+   */
+  shouldComponentRenderHTML () {
+    return !this.render
```

Comment:
isn't there a better way than just a flag to figure out if the html render ran?

Reply:
Nicht dass ich wüsste. Es wird ja nicht unbedingt etwas im HTML hinzugefügt.

Reply:
with querrySelector